website/src/css/custom.css

```
@@ -38,3 +38,7 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
     margin: 0 auto;
   }
 }
+
+.container {
+  max-width: 100vw;
+}
```

I agree that the default max width is not enough for 4k, but the full width is also too large. Let's make a media query to save vertical offsets for content:
```suggestion
@media (min-width: 3160px) {
  .container {
    max-width: 2440px;
  }
}
```